laze-project.yml

```
@@ -424,9 +424,15 @@ modules:
         FEATURES:
           - ariel-os/debug-console
 
+  - name: debug-logging-facade-selection
```

```suggestion
  - name: debug-logging-facade-default
```

(also change were used)

Maybe `-selection` is the better fit, but we`re already using `-default` elsewhere (`executor-default`), so please use that now for consistency and we bikeshed elsewhere!